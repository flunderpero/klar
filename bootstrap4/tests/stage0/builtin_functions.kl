-- Some functions will end up in the prelude eventually.
fn main():
    print(42)
    return 0
end
