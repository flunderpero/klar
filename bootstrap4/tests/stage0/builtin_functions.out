42
---
exit code: 0
